### ISSUE TITLE
Inventory doesn't update

### ISSUE THREAD

**Issue by ks-manu** [tngoman/Store-POS#16]
I downloaded the msi and installed but when I tried some demo purchases the remaining stock didn't change. Basically, if i had 3 cups and I sold 2 the remainder should be one but it remains as 3

**Comment by tngoman** [comment]
Hi, stock changes when you confirm payment, not when you add to cart.

**Comment by ks-manu** [comment]
I did that but the stock wasn't updated. I used the msi link from the readme to download and install. Will recompile and test.
Cheers

**Comment by hamdi-islam** [comment]
@tngoman I can confirm this bug, although it is somehow tricky on how it is occurring. So, in order to reproduce the bug you'll have to put an order on hold first and then go to the "open tabs" and confirm the order. You will notice 2 things:
- The quantity isn't updating for that order
- The quantity won't update on any other order you place until you log out and log back in

**Comment by Ayuen-madyt** [comment]
> @tngoman I can confirm this bug, although it is somehow tricky on how it is occurring. So, in order to reproduce the bug you'll have to put an order on hold first and then go to the "open tabs" and confirm the order. You will notice 2 things:
> 
> * The quantity isn't updating for that order
> * The quantity won't update on any other order you place until you log out and log back in

Yes but I have managed to get fix the code causing the problem. now the inventory updates as it should

**Comment by tngoman** [comment]
Store POS **2.0** (Electron + React + Express + SQLite) replaced the old jQuery / NeDB stack. Please use the current `master` / latest release rather than legacy MSI builds.

Inventory now decrements on paid sales when **Track inventory** is enabled on the product. Re-test on 2.0; reopen with steps if stock still does not move.
